app/views/start.scala.html

```diff
@@ -17,55 +17,51 @@
 @import helpers._
 @import config.ApplicationConfig
 @import uk.gov.hmrc.play.views.html.helpers.form
+@import views.html.mainLayout
+@import views.html._
 
-@(data: Form[String])(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+@this(govukInsetText: GovukInsetText, govukDetails: GovukDetails, mainLayout: mainLayout)
 
+@(data: Form[String])(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
 
-@main_template(
-    title = Messages("start.heading.register"),
-    topNavigation = false
-) {
+@mainLayout(
+pageTitle = messages("start.heading.register"),
+backUri = Some(config.dashboardUrl("home")),
+showTopNavigation = false,
+mainClass = Some("govuk-grid-column-two-thirds")
+){
 
-    <div class="grid-row">
-        <div class="column-two-thirds">
+    <div class="govuk-body">
 
-            <h1 class="heading-xlarge">
+        <h1 class="govuk-heading-xl">
             @messages("start.heading.register")
-            </h1>
-
-            <p>@Html(Messages("start.registration.p.1", routes.Login.show))</p>
-            <p>@Html(Messages("start.registration.p.2", routes.Application.addUserToGG))</p>
-            <p>@messages("start.registration.p.3")</p>
+        </h1>
 
-            <ul class="list list-bullet">
-                <li>@messages("start.registration.youNeed.1")</li>
-                <li>@messages("start.registration.youNeed.2")</li>
-                <li>@messages("start.registration.youNeed.3")</li>
-            </ul>
+        <p class="govuk-body">@Html(Messages("start.registration.p.1", routes.Login.show))</p>
+        <p>@Html(Messages("start.registration.p.2", routes.Application.addUserToGG))</p>
```

Comment:
more p tags that need the govuk class

Reply:
<p> inherits class `govuk-body` from the enclosing `div` - we shouldn't need to add classes to <p> evry time we use them

Reply:
the div wrapper maybe bad practice, so have removed and will add class to <p>'s